### ISSUE TITLE
Erroneous result for offsetHeight on Google Docs

### ISSUE THREAD

**Issue by Ms2ger** [servo/servo#13944]
``` html
.
<script>
    var zK = document.createElement("SPAN");
    zK.style.cssText = "position: absolute; top: -50000px; width: 1in; height: 1in";
    document.body.appendChild(zK);
    console.log(zK.offsetHeight);
    document.body.removeChild(zK);
</script>
```

logs 0 in Servo, but 96 in both Gecko and Chrome.

Google Docs throws an exception ("The number of pixels per point in the user's screen is not set") if the value is falsy ⇒ blocks #13942.

**Comment by Ms2ger** [comment]
Adding `display: block;` helps. CC @pcwalton @mbrubeck

**Comment by jdm** [comment]
This sounds similar to #12939.

**Comment by Permutatrix** [comment]
That's definitely #12939. Sorry for the delay on that again. I'm back now.

**Comment by mbrubeck** [comment]
Fixed by #14839.
